src/_pytest/config/__init__.py

```diff
@@ -1650,6 +1651,66 @@ def getvalueorskip(self, name: str, path=None):
         """Deprecated, use getoption(skip=True) instead."""
         return self.getoption(name, skip=True)
 
+    #: Verbosity for failed assertions (see :confval:`verbosity_assertions`).
+    VERBOSITY_ASSERTIONS: Final = "assertions"
+    _KNOWN_VERBOSITY_TYPES: Final = {VERBOSITY_ASSERTIONS}
+    _VERBOSITY_INI_DEFAULT = "auto"
+
+    def get_verbosity(self, verbosity_type: Optional[str] = None) -> int:
+        r"""Access to fine-grained verbosity levels.
+
+        .. code-block:: ini
+
+            # content of pytest.ini
+            [pytest]
+            verbosity_assertions = 2
+
+        .. code-block:: console
+
+            pytest -v
+
+        .. code-block:: python
+
+            print(config.get_verbosity())  # 1
+            print(config.get_verbosity(Config.VERBOSITY_ASSERTIONS))  # 2
+        """
+        global_level = self.option.verbose
+        assert isinstance(global_level, int)
+        if (
+            verbosity_type is None
+            or verbosity_type not in Config._KNOWN_VERBOSITY_TYPES
+        ):
+            return global_level
+
+        level = self.getini(Config._ini_name(verbosity_type))
+
+        if level == Config._VERBOSITY_INI_DEFAULT:
```

Comment:
The `auto` value should be documented.

Also, why did you choose "auto"?

Reply:
Added some documentation. Also, it was the value used in [this related PR](https://github.com/pytest-dev/pytest/pull/10588/files#diff-0e2d32c88d3fa7edb466f8344dae556d52632e608e7414371f0537ce9dcc3a88R31) on the same topic. I'm open to changing it if there is a better suggestion.